resources/configuration/sites-enabled/tools-context.conf

```diff
@@ -95,6 +72,5 @@ server {
         proxy_redirect off;
         proxy_pass http://ldap-phpadmin;
     }
-    include /etc/nginx/sites-enabled/service-extension/*.conf;
```

Comment:
Why we want to remove this line? It was introduced in https://github.com/Accenture/adop-nginx/pull/22